halyard-config/src/main/java/com/netflix/spinnaker/halyard/model/v1/providers/kubernetes/KubernetesAccount.java

```diff
@@ -32,6 +34,9 @@
   String context;
   String cluster;
   String user;
+  @ValidateField(validators = {ValidateFileExists.class})
+  String kubeconfigFile;
   List<String> namespaces = new ArrayList<>();
+  @ValidateField(validators = {ValidateNotNull.class})
```

Comment:
It would fire if someone tries to delete delete the list of `dockerRegistries` explicitly.